overlay.d/05rhcos/usr/lib/dracut/modules.d/41rhcos-check-luks-syntax/module-setup.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+check() {
+    return 0
+}
+
+install_unit() {
+    local unit="$1"; shift
+    local target="${1:-ignition-complete.target}"; shift
+    local instantiated="${1:-$unit}"; shift
+    inst_simple "$moddir/$unit" "$systemdsystemunitdir/$unit"
+    mkdir -p "$initdir/$systemdsystemunitdir/$target.requires"
+    ln_r "../$unit" "$systemdsystemunitdir/$target.requires/$instantiated"
+}
+
+install() {
+    inst_script "$moddir/rhcos-fail-boot-for-older-luks-syntax" \ 
+        "/usr/sbin/rhcos-fail-boot-for-older-luks-syntax"
```

Comment:
This path should match the one in the service unit (so `/usr/libexec/...`).

Reply:
ah, good catch!